AssetManagementTestProject/TestCases/US303LogoutTest.cs

```diff
@@ -0,0 +1,50 @@
+﻿using NUnit.Framework;
+using NUnit.Framework.Internal;
+using AssetManagementTestProject.PageObj;
+using AssetManagementTestProject.TestSetup;
+
+namespace AssetManagementTestProject.TestCase;
+
+[TestFixture]
+public class US303LogoutTest : NUnitWebTestSetup
+{
+    #region STAFF AND ADMIN LOGOUT
```

Comment:
Why would you need a `#region` if there is only 1 region?

Reply:
I'll fix it